### ISSUE TITLE
Filters are not working for Table Charts

### ISSUE THREAD

**Issue by LiamBAFU** [visualize-admin/visualization-tool#1308]
The Filters you can use for the Table Chart are not working.
On the picture below you can see nothing changed after i applied the filter

![image](https://github.com/visualize-admin/visualization-tool/assets/145760304/42751de1-6b98-4269-81e0-7fe783eefc3b)
![image](https://github.com/visualize-admin/visualization-tool/assets/145760304/434a5033-125f-44cf-958a-73cdfe6894ff)

**To Reproduce**

https://int.visualize.admin.ch/en/v/yRuv98DIMXAd

Open the Link
Make some changes on a Filter in the Table Chart
If you applied the filter theres nothing changing on the chart.

**Expected behavior**
It should apply the Filters on the Chart.

Microsoft Edge
Version 120.0.2210.121 (Offizielles Build) (64-Bit)
v3.25.0 (1b7a837)

**Comment by ortnever** [comment]
@bprusinowski 
Dear Bartosz, it looks better on Test : Filters work for Table. But why isn't possible to activate "Interactiv Filter" for Table?

**Comment by bprusinowski** [comment]
Hi @ortnever, thanks for noticing the problem with interactive filters, I am reopening the issue 👀

**Comment by bprusinowski** [comment]
Hi @ortnever, I took a look and it looks like we have never supported interactive filters with tables, it's also not possible to enable them on PROD currently.

I removed the "Interactive" toggle from temporal dimensions used in tables, as it doesn't have any effect on the chart; additionally, I noticed that select elements are empty when creating a new table, like for [this cube](https://visualize.admin.ch/en/create/new?cube=https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/9) (see the Kanton column once switching to table). These things will be available on TEST (and INT) once we merge https://github.com/visualize-admin/visualization-tool/pull/1351.

I think it would make sense to open a new issue to enable interactive filters for table, as it doesn't seem to be a regression but rather a missing feature.

cc @sosiology
